sdcv.el

```diff
@@ -507,6 +510,9 @@ The result will be displayed in buffer named with
 
 (defun sdcv-search-simple (&optional word)
   "Search WORD simple translate result."
+  (unless (featurep 'posframe)
```

Comment:
Use 

```
(when (igonre-errors (require 'posframe))
   ...
```

Instead featurep, because user won't require posframe even he has install it.

Reply:
I saw that `posframe-show` is an autoloaded function, so I thought that emacs will load it automatically. Of course, the solution with `require` will make it more robust.

Reply:
Don't need autload.